frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -283,12 +291,12 @@ const PropertyListView: React.FC = () => {
           </div>
           <TooltipWrapper toolTipId="export-to-excel" toolTip="Export to Excel">
             <FileIcon>
-              <FaFileExcel size={36} onClick={() => fetch('excel')} />
+              <FaFileExcel size={36} title="Export to Excel" onClick={() => fetch('excel')} />
             </FileIcon>
           </TooltipWrapper>
           <TooltipWrapper toolTipId="export-to-excel" toolTip="Export to CSV">
             <FileIcon>
-              <FaFileAlt size={36} onClick={() => fetch('csv')} />
+              <FaFileAlt size={36} title="Export to CSV" onClick={() => fetch('csv')} />
```

Comment:
I'd refrain from having the title and the tooltip.  I prefer the tooltip because you have more control.

Reply:
Fixing this right now!